.github/workflows/update-major-and-minor-version.yml

```diff
@@ -0,0 +1,39 @@
+# Trigger the workflow after publishing releases
+name: Update major and minor versions
+run-name: Move ${{ github.event.inputs.major_version }} to ${{ github.event.inputs.target }}
+
+on:
+  workflow_dispatch:
```

Comment:
It looks manual. Can we automate it?

Reply:
haha, I would call it "semi-auto". But yes, looking for a fully auto method to do this. If I cannot find a good way today, I will just remove it from this iteration. @shizhMSFT

Reply:
Updated to fully automate version.